k8s/aat/common/bsp/bulk-scan-payment-processor.yaml

```diff
@@ -22,8 +29,33 @@ spec:
       useInterpodAntiAffinity: true
       applicationInsightsInstrumentKey: "f666440f-43c0-4abb-9c20-2ce175a374f0"
       image: hmctspublic.azurecr.io/bulk-scan/payment-processor:prod-d94753df
-      environment:
-        RESTART_ME: "Update due to vault change"
+      testsConfig:
+        memoryLimits: "3072Mi"
+        serviceAccountName: tests-service-account
+        keyVaults:
+          bulk-scan:
+            resourceGroup: bulk-scan
+            secrets:
+              s2s-secret-payment-processor: S2S_SECRET
+              idam-users-bulkscan-username: IDAM_USERS_BULKSCAN_USERNAME
+              idam-users-bulkscan-password: IDAM_USERS_BULKSCAN_PASSWORD
+              idam-client-secret: IDAM_CLIENT_SECRET
+              payments-queue-send-connection-string: PAYMENTS_QUEUE_WRITE_CONNECTION_STRING
+        environment:
+          TEST_URL: "http://bulk-scan-payment-processor-aat.service.core-compute-aat.internal"
+          SLACK_CHANNEL: "bsp-test-notices"
+          SLACK_NOTIFY_SUCCESS: true
+          S2S_NAME: "bulk_scan_payment_processor"
+          S2S_URL: "http://rpe-service-auth-provider-aat.service.core-compute-aat.internal"
+          CORE_CASE_DATA_API_URL: "http://ccd-data-store-api-aat.service.core-compute-aat.internal"
+          IDAM_API_URL: "https://idam-api.aat.platform.hmcts.net"
+          IDAM_CLIENT_REDIRECT_URI: "https://bulk-scan-orchestrator-aat.service.core-compute-aat.internal/oauth2/callback"
```

Comment:
do we need the redirect url for smoke/functional ? I thought only app needs it ?

Reply:
Functional tests use Idam client for talking to CCD.